opm/parser/eclipse/EclipseState/IOConfig/RestartConfig.cpp

```diff
@@ -360,6 +360,24 @@ RPTSCHED( const DeckKeyword& keyword ) {
 }
 
 
+    void RestartConfig::addRestartKeywords( size_t timeStep , const std::map<std::string, int>& input_mnemonics) {
```

Comment:
Hehe - as you say MMV; personally I maintain a good-treatment-of-datastructures ordering which looks like this:

1. Immutable - best.
2. Add elements - middle.
3. Remove elements - worst.

Anyway - I'll look into the possibility of a hybrid approach.